kratos/tests/containers/test_model.cpp

```diff
@@ -0,0 +1,75 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Philipp Bucher
+//
+//
+
+
+// Project includes
+#include "testing/testing.h"
+#include "containers/model.h"
+
+namespace Kratos {
+	namespace Testing {
+
+		KRATOS_TEST_CASE_IN_SUITE(ModelGetModelPart, KratosCoreFastSuite)
+		{
+            auto model_part = Kratos::make_shared<ModelPart>("Main");
+
+			model_part->CreateSubModelPart("Inlet1");
+
+            Model model;
+
+            model.AddModelPart(model_part);
+
+            KRATOS_CHECK_EQUAL(model.GetModelPart("Main"), *model_part);
+
+            ModelPart& smp = model_part->GetSubModelPart("Inlet1");
+            KRATOS_CHECK_EQUAL(model.GetModelPart("Main.Inlet1"), smp);
```

Comment:
is this correct. It works for me, but I am not sure if this comparison is ok

Reply:
i would check for the Name()